### PR TITLE
WPF 886828 - Updated Selection mode topic changes

### PR DESCRIPTION
**Pull request by TamilarasanGunasekaran**
I have updated the Selection mode topic changes.

### CODE REVIEW

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 1]
```diff
@@ -19,7 +19,6 @@ The following codes illustrate this
 …
 <gantt:GanttControl x:Name="Gantt"
                     ItemsSource="{Binding TaskDetails}"
-                    VisualStyle="Metro"
                     Loaded="Gantt_Loaded">
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:22]: Rename to `OnGanttLoaded`

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Restrict SelectionMode as Single in WPF Gantt
 
-To restrict the selection mode to single, set the ListBoxSelectionMode options of GanttGrid’s Model in the GanttControl loaded event. 
+To restrict the selection mode to single, set the [SelectionMode](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttControl.html#Syncfusion_Windows_Controls_Gantt_GanttControl_SelectionMode) of Gantt in the [GanttControl](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttControl.html) loaded event.
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:12]: @TamilarasanGunasekaran , Check in Vol 1 2024, release version with existing code changes and confirm.

**Review by Karthickmani97 — COMMENTED** [review]
@TamilarasanGunasekaran , Address my review corrections.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 1 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md]: Renamed.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 2 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:12]: The existing code is working fine in Vol 1 2024, release version.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 3]
```diff
@@ -1,38 +1,163 @@
 ---
 layout: post
-title: Restrict SelectionMode as Single | Gantt | Wpf | Syncfusion
-description: This section describes how to restrict the SelectionMode behavior as Single in GanttControl for WPF platform.
+title: Restrict SelectionMode | Gantt | Wpf | Syncfusion
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:3]: Restrict item selection | Gantt | Wpf | Syncfusion

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 4]
```diff
@@ -1,38 +1,163 @@
...
+description: This section describes how to restrict the SelectionMode behavior in Gantt control for WPF platform.
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:4]: This section describes how to implement the SelectionMode behavior in the WPF Gantt Control.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 5]
```diff
@@ -1,38 +1,163 @@
...
 platform: wpf
 control: Gantt
 documentation: ug
 ---
 
-# Restrict SelectionMode as Single in WPF Gantt
+# Restrict SelectionMode
 
-To restrict the selection mode to single, set the ListBoxSelectionMode options of GanttGrid’s Model in the GanttControl loaded event. 
+To restrict the selection mode, set the [SelectionMode](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttControl.html#Syncfusion_Windows_Controls_Gantt_GanttControl_SelectionMode) of the [GanttControl](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttControl.html) as None.
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:12]: To restrict item selection, set the `SelectionMode` of the `GanttControl` to `None.`

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 6]
```diff
@@ -1,38 +1,163 @@
...
 
 {% tabs %}
 {% highlight xaml %}
 
-…
 <gantt:GanttControl x:Name="Gantt"
                     ItemsSource="{Binding TaskDetails}"
-                    VisualStyle="Metro"
-                    Loaded="Gantt_Loaded">
+                    SelectionMode="None">
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:21]: use Code highlight for this SelectionMode

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 7]
```diff
@@ -1,38 +1,163 @@
...
 {% highlight c# %}
 
-…
-private void Gantt_Loaded(object sender, RoutedEventArgs e)
-{
-    this.Gantt.GanttGrid.Model.Options.ListBoxSelectionMode = Syncfusion.Windows.Controls.Grid.GridSelectionMode.One;
-}
-…
+this.ganttControl.ItemsSource = new ViewModel().TaskCollection;
+this.ganttControl.SelectionMode = GanttSelectionMode.None;
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:42]: Highlights code

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 8]
```diff
@@ -1,38 +1,163 @@
...
-description: This section describes how to restrict the SelectionMode behavior as Single in GanttControl for WPF platform.
+title: Restrict SelectionMode | Gantt | Wpf | Syncfusion
+description: This section describes how to restrict the SelectionMode behavior in Gantt control for WPF platform.
 platform: wpf
 control: Gantt
 documentation: ug
 ---
 
-# Restrict SelectionMode as Single in WPF Gantt
+# Restrict SelectionMode
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:10]: Restrict item selection

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 9]
```diff
@@ -1,38 +1,163 @@
...
-…
-private void Gantt_Loaded(object sender, RoutedEventArgs e)
-{
-    this.Gantt.GanttGrid.Model.Options.ListBoxSelectionMode = Syncfusion.Windows.Controls.Grid.GridSelectionMode.One;
-}
-…
+this.ganttControl.ItemsSource = new ViewModel().TaskCollection;
+this.ganttControl.SelectionMode = GanttSelectionMode.None;
+
+// Task attribute mapping
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:44]: Remove mapping codes as not required for default models in both XAML and C#.

**Review by Karthickmani97 — COMMENTED** [review]
@TamilarasanGunasekaran , Address my review concerns.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 3 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md]: Changed.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 4 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md]: Changed.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 5 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md]: Changed.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 6 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:21]: Highlighted.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 7 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:31]: Highlighted.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 8 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md]: Changed.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 9 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md]: Removed.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 10]
```diff
@@ -1,38 +1,139 @@
...
+To restrict item selection, set the [SelectionMode](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttControl.html#Syncfusion_Windows_Controls_Gantt_GanttControl_SelectionMode) of the [GanttControl](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttControl.html) to None. 
 
 The following codes illustrate this
 
 {% tabs %}
-{% highlight xaml %}
+{% highlight xaml hl_lines="3" %}
 
-…
 <gantt:GanttControl x:Name="Gantt"
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:19]: rename to `gantt`

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 11]
```diff
@@ -1,38 +1,139 @@
...
-{% highlight c# %}
+{% highlight c# hl_lines="2" %}
 
-…
-private void Gantt_Loaded(object sender, RoutedEventArgs e)
-{
-    this.Gantt.GanttGrid.Model.Options.ListBoxSelectionMode = Syncfusion.Windows.Controls.Grid.GridSelectionMode.One;
-}
-…
+this.ganttControl.ItemsSource = new ViewModel().TaskCollection;
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:30]: use `this. gantt`

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md [hunk 12]
```diff
@@ -1,38 +1,139 @@
...
 platform: wpf
 control: Gantt
 documentation: ug
 ---
 
-# Restrict SelectionMode as Single in WPF Gantt
+# Restrict item selection
 
-To restrict the selection mode to single, set the ListBoxSelectionMode options of GanttGrid’s Model in the GanttControl loaded event. 
+To restrict item selection, set the [SelectionMode](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttControl.html#Syncfusion_Windows_Controls_Gantt_GanttControl_SelectionMode) of the [GanttControl](https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttControl.html) to None.
```
Comment (Karthickmani97) [inline wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md:12]: Set this link to `None`

https://help.syncfusion.com/cr/wpf/Syncfusion.Windows.Controls.Gantt.GanttSelectionMode.html#Syncfusion_Windows_Controls_Gantt_GanttSelectionMode_None

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 10 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md]: Renamed.

File: wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md (hunk 11 above)
Comment (TamilarasanGunasekaran) [reply wpf/Gantt/How-To/Restrict-SelectionMode-as-Single.md]: Changed.

**Review by Karthickmani97 — APPROVED** [review]
This is fine.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**